packages/frontend-shared/src/components/Tooltip.vue

```diff
@@ -16,9 +18,11 @@ import { Tooltip } from 'floating-vue'
 withDefaults(defineProps<{
   color?: string
   hideArrow?: boolean
+  skipTooltip?: boolean
```

Comment:
Nit: thoughts on calling this prop `disabled` to align with the standard semantics?

My initial thought is "Why are we rendering a tooltip if we don't want to show a tooltip?" But I assume it's because the vue templates aren't flexible enough to reuse/dry up complex child markup. We could move to a render function to better enable that, but that doesn't sound ideal either.

Reply:
Name updated